src/components/analytics/preview.vue

```diff
@@ -146,8 +165,11 @@ export default {
     "projects": {
       // This is the title shown above a series of metrics about Project usage.
       "title": "Project Summaries",
-      "subtitle": "Showing 1 Project of {numProjects}"
-    }
+      "subtitle": "(Showing the most active Project of {numProjects} Projects)"
```

Comment:
With this rewording, I think this does need pluralization now. Even in English, there's the possibility of there being just one project, in which case I think we should show "Project" at the end instead of "Projects". Vue I18n has good docs around pluralization, but the upshot is that this message will change to:

```
"subtitle": "(Showing the most active Project of {count} Project) | (Showing the most active Project of {count} Projects)"
```

Notice that `{numProjects}` becomes `{count}`. That's because Vue I18n passes in the count as a variable named `{count}`.

Other changes:

```diff
-<span class="explanation">{{ $t('projects.subtitle', { numProjects }) }}</span>
+<span class="explanation">{{ $tcn('projects.subtitle', numProjects) }}</span>
```

```diff
 numProjects() {
-  return this.$n(this.analyticsPreview.projects.length);
+  return this.analyticsPreview.projects.length;
 }
```

We no longer call `this.$n()` in `numProjects()`, because Vue I18n needs a number, not a localized string, in order to pluralize. However, `$tcn()` (a little utility I wrote) will call `$n()` on the count before passing it to the message via `$tc()`.

By the way, I don't think the test will need to change at all.

Reply:
I added a single project test as well!